### PR TITLE
Echo version string into console after build completion

### PR DESCRIPTION
**Pull request by neomatrix369**
- check if the build has produced the JDK image in the right folder and contains the `java` launcher
- run the `java -version` command to produce the version string onto the console
- fixed the issue with `BUILD_FULL_NAME` not being passed into all the sub-scripts

Created on the back of PRs #93, #97 and #101

### CODE REVIEW

**Comment by neomatrix369** [comment]
Kicked off build https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk_build_ppc64le_linux_mani_test/3/console - sanity check if the string appears in the console as expected

File: sbin/common-functions.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ determineBuildProperties() {
 
     BUILD_TYPE=normal
     DEFAULT_BUILD_FULL_NAME=${OS_KERNEL_NAME}-${OS_MACHINE_NAME}-${BUILD_TYPE}-${JVM_VARIANT}-release
-    BUILD_FULL_NAME=${BUILD_FULL_NAME:-"$DEFAULT_BUILD_FULL_NAME"}
+    export BUILD_FULL_NAME=${BUILD_FULL_NAME:-"$DEFAULT_BUILD_FULL_NAME"}
```
Comment (neomatrix369) [inline sbin/common-functions.sh]: Updating the env variable here I guess this isn't a bad thing - this has gone unnoticed i think hope other types of builds work fine

**Comment by neomatrix369** [comment]
When successful on a platform, prints this at the bottom of the logs:
```
'/home/jenkins/workspace/openjdk_build_x86-64_linux_johno_test/openjdk/build/linux-x86_64-normal-server-release/images/j2sdk-image' found

openjdk version "1.8.0_152-adoptopenjdk"
OpenJDK Runtime Environment (build 1.8.0_152-adoptopenjdk-builddateb00-b00)
OpenJDK 64-Bit Server VM (build 25.152-b00, mixed mode)
```

**Comment by neomatrix369** [comment]
Merged changes to version string by @karianna via PR #101

**Comment by neomatrix369** [comment]
https://ci.adoptopenjdk.net/view/work%20in%20progress/ - most builds are successful (ignore Windows as it does not yet use the makejdk scripts). Two builds still in progress.

**Comment by neomatrix369** [comment]
All checks have failed -  shellcheck failiures due known reasons

File: sbin/build.sh [hunk 2]
```diff
@@ -197,6 +197,23 @@ buildOpenJDK()
   echo "${normal}"
 }
 
+
+printJavaVersionString()
```
Comment (karianna) [inline sbin/build.sh]: some white space indentation issues in this function?

File: sbin/common-functions.sh (hunk 1 above)
Comment (karianna) [inline sbin/common-functions.sh]: I assume this is the only place it is declared?

Comment (neomatrix369) [inline sbin/common-functions.sh]: One of the entry points - will check, fairly sure it does not affect anything, all the builds I ran this morning across all our platforms work fine despite this change, it takes the existing value and updates the env variable

Comment (neomatrix369) [inline sbin/common-functions.sh]: Just checked `makejdk-any-platform.sh` is the entry point, but if its not set there, it never gets set, so `export` keyword is important for setting a default value

**Pull request merged**